modules/exploits/multi/http/motioneye_auth_rce_cve_2025_60787.rb

```diff
@@ -0,0 +1,404 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Remote Code Execution Vulnerability in MotionEye (CVE-2025-60787)',
+        'Description' => %q{
+          MotionEye is vulnerable to OS Command Injection in configuration parameters such as image_file_name.
+          Unsanitized user input is written to Motion configuration files,
+          allowing remote authenticated attackers with admin access to achieve code execution when Motion is restarted.
+
+          This vulnerability affects MotionEye versions <= 0.43.1b4 are vulnerable.
```

Comment:
```suggestion
```

Remove this and add to first line of description per above comment.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/20585/commits/25624e9ae0cd5a456752aae337bb12825b5b4322